public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
- [ ] [Optional][Highly recommended] It would be better professionally if you remove all boilerplate files from your app.
You will find them marked in the image below.

![image](https://github.com/danifromecuador/bookstore/assets/68796232/8678fcc5-5a30-4d54-83d5-5fa02d4b3081)

Reply:
Thank you for your detailed suggestion @Omar-Muhamad